### PR TITLE
Update abbreviations.rst

### PR DESCRIPTION
**Pull request by walterkozlowski**
Updating Abbreviation based on latest updates to RM

### CODE REVIEW

**Comment by walterkozlowski** [comment]
@jbhartley, @karinesevilla, @pgoyal01, @ulikleber - please review and approve if you can

File: doc/common/abbreviations.rst [hunk 1]
```diff
@@ -260,12 +272,16 @@ PTP            Precision Time Protocol
 PV             Persistent Volumes
 PVC            Persistent Volume Claims
 PXE            Preboot Execution Environment
+QCW            QEMU copy-on-write
 QEMU           Quick EMUlator
 QoS            Quality of Service
 R/W            Read/Write
 RA             Reference Architecture
+RADOS          Reliable Autonomic Distributed Object Store
+RAID           Redundant Aray of Independent Disks
```
Comment (ulikleber) [inline doc/common/abbreviations.rst:281]: typo
```suggestion
RAID           Redundant Array of Independent Disks
```

File: doc/common/abbreviations.rst [hunk 2]
```diff
@@ -390,4 +408,5 @@ Wi-Fi          Wireless Fidelity
 WLAN           Wireless Local Area Network
 WLC            Wireless LAN Controller
 WS             WorkStream
-============== ==========================================================================
+XML            eXtensible Markup Language
+========================================================================================
```
Comment (ulikleber) [inline doc/common/abbreviations.rst:412]: You shouldn't remove the blank 
```suggestion
============== ==========================================================================
```

File: doc/common/abbreviations.rst [hunk 3]
```diff
@@ -260,12 +272,16 @@ PTP            Precision Time Protocol
 PV             Persistent Volumes
 PVC            Persistent Volume Claims
 PXE            Preboot Execution Environment
+QCW            QEMU copy-on-write
```
Comment (collivier) [inline doc/common/abbreviations.rst:275]: QCOW

File: doc/common/abbreviations.rst [hunk 4]
```diff
@@ -136,11 +143,14 @@ HLR            Home Location Register
 HOT            (OpenStack) Heat Orchestration Templates
 HSS            Home Subscriber Server
 HTML           Hyper Text Markup Language
+HTTP           Hypertext Transfer Protocol
+HTTP/S         Hypertext Transfer Protocol Secure
```
Comment (collivier) [inline doc/common/abbreviations.rst:147]: why not HTTPS ?

**Comment by walterkozlowski** [comment]
@pgoyal01, accpeted proposed changes, thanks.  Can you approve now?
@collivier, are you going to fix the build errors?  Thanks.

**Review by pgoyal01 — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Comment by CsatariGergely** [comment]
There are several error causing the biuld to fail: 
./chapters/chapter04.rst:446: D000 Malformed table.
./chapters/chapter04.rst:992: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter04.rst:1089: D002 Trailing whitespace
./chapters/chapter02.rst:865: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter02.rst:881: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter02.rst:899: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter02.rst:908: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter02.rst:333: D002 Trailing whitespace
./chapters/chapter02.rst:435: D002 Trailing whitespace
./chapters/chapter03.rst:262: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:262: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:262: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:262: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:290: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:578: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:578: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:578: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:829: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:829: D000 Inline interpreted text or phrase reference start-string without end-string.
./chapters/chapter03.rst:290: D002 Trailing whitespace
These should be fixed, but I would propose an other pr to keep the concerns separated.

**Comment by pgoyal01** [comment]
@CsatariGergely I opened an Issue for each Chapter -- currently assigned to @rgstori

**Pull request merged**